### PR TITLE
Updating C5G7 tests. Adding weekly test workflow.

### PR DESCRIPTION
**Pull request by wdhawkins**
This PR fixes an issue with the test script for problems that support additional lua arguments. It also adds a github actions workflow for long-running/moderately-sized weekly tests.

### CODE REVIEW

File: test/modules/linear_boltzmann_solvers/transport_keigen/c5g7/c5g7_jfnk.lua [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+        power_normalization = 1.0,
+        save_angular_flux = true
+    },
+    sweep_type = "AAH"
+})
+
+-- Execute Solver
+if (k_method == "pi") then
+    k_solver = lbs.XXPowerIterationKEigen.Create
+    ({
```
Comment (ragusa) [inline test/modules/linear_boltzmann_solvers/transport_keigen/c5g7/c5g7_jfnk.lua:47]: still XX in name?

**Review by ragusa — APPROVED** [review]

**Comment by ragusa** [comment]
@wdhawkins I am curious on the compute time between CBC and AAH. Can you tell me the difference, when running on the same machine? Thanks

File: test/modules/linear_boltzmann_solvers/transport_keigen/c5g7/c5g7_jfnk.lua (hunk 1 above)
Comment (wdhawkins) [reply test/modules/linear_boltzmann_solvers/transport_keigen/c5g7/c5g7_jfnk.lua:47]: Yes, the other PR hasn't been merged yet. When it is, I'll rebase and update this one.

**Comment by wdhawkins** [comment]
> @wdhawkins I am curious on the compute time between CBC and AAH. Can you tell me the difference, when running on the same machine? Thanks

For PI, AAH is ~30% faster than CBC (200 ranks, Cascade Lake, OmniPath). CBC takes 132 its. AAH takes 173 its with 2% lagged unknowns. For JFNK, CBC is ~20% faster than AAH. CBC takes 7 outers  vs 19 for AAH.

**Review by andrsd — APPROVED** [review]

**Review by zhardy-lanl — CHANGES_REQUESTED** [review]
It looks like each of the test files are identical and that the only difference is that in the .json file you specify the `k_method` argument. Do we need a file for each if this is the case?

**Comment by wdhawkins** [comment]
> It looks like each of the test files are identical and that the only difference is that in the .json file you specify the `k_method` argument. Do we need a file for each if this is the case?

Yeah, not a fan of this either. Unfortunately, it looks like the test script requires a unique file name for each test... even if the Lua args differ. I'll open an issue for this problem. In the meantime, I wanted to get these tests running, so I just created a separate input for each case.

**Review by zhardy-lanl — APPROVED** [review]

**Pull request merged**